assignments/testdata/tests/lab4/criteria.json

```diff
@@ -0,0 +1,10 @@
+[
+    {
+        "CourseID": 0,
```

Comment:
To best match how we use the `criteria.json` file "in the wild" you could omit the `CourseID` here (and below on line 6).

Example of how a teacher would create `criteria.json` (no course ID):

```json
[
    {
        "heading": "Assignment 1",
        "criteria": [
            {
                "description": "Links work"
            },
            {
                "description": "Images are links, opening in a new tab"
            }
        ]
    }
]
```

The minimal `criteria.json` file we could use in this test is

```json
[
    {
        "criteria": [
            {}
        ]
    }
]
```

Reply:
@JoachimTislov I suggest to add more `criteria.json` example files to test different legal versions. You may need to add more `labX` folders to `testdata/tests/`... since you cannot have multiple `criteria.json` files per lab.

Also, you may consider adding an invalid `criteria.json` file to check that bad files give an expected parse error, but I think it is enough to provide one example.